src/hotspot/cpu/zero/zeroInterpreter_zero.cpp

```diff
@@ -619,6 +619,8 @@ int ZeroInterpreter::getter_entry(Method* method, intptr_t UNUSED, TRAPS) {
       stack->alloc(wordSize);
       topOfStack = stack->sp();
       break;
+    default:
+      ;
```

Comment:
> `ShouldNotReachHere()` ? Maybe an `assert()` ?

We should not add `ShouldNotReachHere()` or `assert()` here.
This is because the `entry->flag_state()` may be `itos/ftos...`.
The logic is only add one more slot for long/double.
So just do nothing here.

Reply:
Eww, OK.